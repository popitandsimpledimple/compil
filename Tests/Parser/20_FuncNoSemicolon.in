function abc(var a: Double; const b: Integer; c: String)
begin end;

begin
end.