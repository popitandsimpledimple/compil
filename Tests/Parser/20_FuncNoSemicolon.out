(2,1) Expected 'Colon' Found 'BEGIN'